src/tui.rs

```diff
@@ -93,6 +93,7 @@ macro_rules! funny_opening {
             "I scanned ports so fast, even my computer was surprised.",
             "RustScan: Where '404 Not Found' meets '200 OK'.",
             "RustScan: Exploring the digital landscape, one IP at a time.",
+            "DeathStarMafia was here 🚀",
```

Comment:
question: is this suitable?

One for @bee-san: is this okay to add? Not sure there are other, named individuals/orgs. in this list…?

Reply:
Lol that's my CTF team (I did not name it), but I'll change this. But yes the 0day guy added himself to this but maybe he's an exception as he's tiktok famous

Reply:
I didn't realise "0day" was a person; thought it was a reference to the _thing_. Shows what I know.

Kudos on the CTF team name though.

Reply:
@PsypherPunk 
@BrendanGlancy yes, 0day is one of my friends. his dog passed away sadly and i wanted to commemorate him (Ollie), 0day put in a PR to add his own name so I thought it'd be ok 😅 

@BrendanGlancy I will let you add your name here if you add 3 more networking related / hacking jokes 😉